proto/servicemanager/v3/servicemanager.proto

```diff
@@ -0,0 +1,297 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package servicemanager.v3;
+
+/***********************************************************************************************************************
+ * SMService
+ **********************************************************************************************************************/
+
+service SMService {
+    rpc RegisterSM(stream SMNotifications) returns (stream CMRequests) {}
```

Comment:
Having requests -> responses would be more logical. Let's rename `SMNotifications` -> `SMResponses`

Reply:
renamed to SMOutgoingMessages